packages/core/src/Stores/traders-hub-store.js

```diff
@@ -878,4 +879,10 @@ export default class TradersHubStore extends BaseStore {
     setIsSetupRealAccountOrGoToDemoModalVisible(value) {
         this.is_setup_real_account_or_go_to_demo_modal_visible = value;
     }
+
+    get regulated_mt5_accounts() {
```

Comment:
Isn't it supposed to be an API flag? Why do we need to add business logic in the FE code? 🫨

Reply:
This is a work around to compute if TIN is mandatory. During the development of this feature, we didn't consider the behavior of Verify Phone number button to also depend on the whole form submission